### DISCUSSION TITLE
What version of template am I using?

### DISCUSSION THREAD

**Discussion by rebullm** [greenelab/lab-website-template#184] [General] [answered]
Hi- Can you tell me where to find the version of the template I am using and what is the current released version? 
Thanks!

**Comment by vincerubinetti** [comment, marked as answer]
The current released version of the template is on its [repo homepage](https://github.com/greenelab/lab-website-template) on the side:

<img width="584" alt="Screenshot 2023-04-17 at 11 30 25 AM" src="https://user-images.githubusercontent.com/8326331/232536005-666fa636-0b4b-4d89-b53f-cc733bb4fe1b.png">

Your version of the template is in your `CITATION.cff` file.
